src/spikeinterface/qualitymetrics/tests/conftest.py

```diff
@@ -5,6 +5,8 @@
     create_sorting_analyzer,
 )
 
+job_kwargs = dict(n_jobs=2, progress_bar=True, chunk_duration="1s")
```

Comment:
Is there a reason we do 2 instead of -1 to just see what the runners allow? I honestly don't know was just curious?

Reply:
Let's try it out...

Reply:
Seems like it is roughly the same. I think the wrapping in main was the big thing for Windows so kudos!

Reply:
Yup, pretty similar: should we stick with `-1`?

Reply:
Unless there was a reason not to? Maybe there was a concern of stressing the runner? That's before my time why that choice would be made. I do know we had a test that n=4 and one of the mac runners only had 3 cores available at one time which caused stuff to fail, so I feel like -1 is better because it auto-adjust (like if they give us a single core runner 2 might mess up). But I don't know if I have the authority to say definitely stick with -1....?

Reply:
A reason not to: we're forcing the user to use `-1`. So if I ran these tests on my cluster, it'd go WILD (it thinks it has access to _all_ the cores on the cluster, which it does not!). If the time difference is minimal, it's safer to keep `2`?

Reply:
I bet that was the reason. It was to protect the local testing from it doing all cores.